homeassistant/components/group/registry.py

```diff
@@ -70,7 +75,8 @@ def on_off_states(
             if on_state not in self.on_off_mapping:
                 self.on_off_mapping[on_state] = off_state
 
-        if len(on_states) == 1 and off_state not in self.off_on_mapping:
+        if off_state not in self.off_on_mapping:
```

Comment:
I think we should extend the module docstring and maybe this method docstring and say that we won't add support for new domains to the legacy groups.

Reply:
Also added a docs PR to make the supported domains explict. In a follow PR We can auto exclude other domains.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/32574